public/app/view.js

```diff
@@ -15,10 +15,25 @@ angular.module('app.view', [])
   // make editor read only
   editor.setReadOnly(true);
 
+  $scope.editRoute = function (id) {
```

Comment:
When I test, I'm getting redirected to /main. Maybe something to do with this function expecting an but it not getting one passed?